handlers/commands/groups.js

```diff
@@ -16,13 +18,25 @@ const entry = group => group.username
 	? `- @${group.username}`
 	: `- <a href="${group.link}">${escapeHtml(group.title)}</a>`;
 
+const emojiRegex = XRegExp.tag('g')`
+	[\uE000-\uF8FF]|
```

Comment:
Line breaks here break the regex and `stripEmoji` doesn't strip the emojis. However, it works without line breaks.